dags.yaml

```diff
@@ -978,3 +978,25 @@ bqetl_fivetran_costs:
   schedule_interval: 0 5 * * *
   tags:
     - impact/tier_3
+
+bqetl_mdn_yari:
+  default_args:
+    depends_on_past: false
+    email:
+      - telemetry-alerts@mozilla.com
+      - mdn@mozilla.com
+      - fmerz@mozilla.com
+    email_on_failure: true
+    email_on_retry: false
+    end_date: null
+    owner: fmerz@mozilla.com
```

Comment:
We should have someone from Data Team to either own this or be on the `email` list as support.

Reply:
Added myself to email list.